src/public/config.json

```diff
@@ -14,7 +14,7 @@
   "pagination_page_size": 25,
   "postcode_lookup_url": "http://localhost:3453/addresses?postcode=${postcode}",
   "print_service_url": "http://localhost:3453/print",
-  "remote_document_management_url": "https://api-gateway.dev.dm.reform.hmcts.net/documents",
+  "remote_document_management_url": "http://localhost:4603/documents",
```

Comment:
Is this intentional?

Reply:
This is for working with local DM store instance, without this the document URLs are broken. This parameter is overridden in environments anyway...

Reply:
Ok fair enough.